cpp/include/cudf/strings/padding.hpp

```diff
@@ -28,10 +28,10 @@ namespace strings
  * @brief Pad types for the pad method specify where the pad
  * character should be placed.
  */
-enum class pad_side {
-    left,   ///< Add padding to the left.
-    right,  ///< Add padding to the right.
-    both    ///< Add padding equally to the right and left.
+enum pad_side {
```

Comment:
What was the reason for removing class ?

Reply:
Good catch @rgsl888prabhu, reverted this and made appropriate changes in the corresponding `.pxd`.